sdk/iot/Azure.Iot.Hub.Service/samples/IotHubClientSamples/ModuleIdentityLifecycleSamples.cs

```diff
@@ -109,7 +107,7 @@ public async Task<ModuleIdentity> CreateModuleIdentityAsync(DeviceIdentity devic
                 // Call APIs to create the module identity.
                 Response<ModuleIdentity> response = await IoTHubServiceClient.Modules.CreateOrUpdateIdentityAsync(moduleIdentity);
 
-                SampleLogger.PrintSuccess($"Successfully created a new module identity with Id: '{moduleId}'");
+                SampleLogger.PrintSuccess($"Successfully created a new module identity: DeviceId: '{deviceId}', ModuleId: '{moduleId}', ETag: '{response.Value.Etag}'");
```

Comment:
nit - how do you feel about printing the device Id values from the `response.Value`? I now that it'll be the same as what we pass in, and we are not validating the service here, but from a sample's perspective, would it make more sense to print the values from the returned object?